### DISCUSSION TITLE
Tips dan trick agar bisa remote work di upwork / lainnya

### DISCUSSION THREAD

**Discussion by teguhkrniawan** [deaafrizal/forum#4] [The Lounge]
Izin bg dea, buat diskusi, mumpung masih baru ni forum githubnya hehe. Boleh dong berbagi pengalaman dari temen2 yang sudah tembus kerja as remote worker, apa ajasih yang harus disiapakan agar bisa dihire. ex seperti portofolio yang gimana sih yang dilirik oleh orang-orang diluar negeri ? Terimakasih.

**Comment by deaafrizal** [comment]
noted bro, kita tunggu kali aja ada yang sharing dari yang udah pernah remote upwork ya 👌

**Reply by teguhkrniawan** [reply]
siap bang

**Reply by pikupik** [reply]
Iyanih, baru mau mulai ngambil projectan remote entah itu di freelancer upwork dll, minta tipsnya dong buat yang tau

**Reply by teguhkrniawan** [reply]
sepertinya belum dapat titik terang ya mas wwkwkwk

**Reply by deaafrizal** [reply]
gue sih gak pernah sengaja ngajuin, rata2 projek di rekomendasiin ama relasi2 gue 😄 jadi gue gak bisa banyak komen ini mah haha

**Reply by teguhkrniawan** [reply]
siap bang aman, paham kok, sebelumnya bang dea udah pernah bahas ini juga di channel bang dea
